src/arfsdao.ts

```diff
@@ -709,6 +709,21 @@ export class ArFSDAO extends ArFSDAOAnonymous {
 		return latestRevisionsOnly ? allFiles.filter(latestRevisionFilter) : allFiles;
 	}
 
+	async getChildrenFolderIds(
+		folderId: FolderID,
+		allFolderEntitiesOfDrive: ArFSPublicFolder[] | ArFSPrivateFolder[]
+	): Promise<FolderID[]> {
+		const hierarchy = FolderHierarchy.newFromEntities(allFolderEntitiesOfDrive);
+		return hierarchy.folderIdSubtreeFromFolderId(folderId, Number.POSITIVE_INFINITY);
```

Comment:
`maxDepth` should'n be greater than `Number.MAX_SAFE_INTEGER`
```suggestion
		return hierarchy.folderIdSubtreeFromFolderId(folderId, Number.MAX_SAFE_INTEGER);
```

Reply:
The algorithm right now is safe because:
```ts
maxDepth = Number.POSITIVE_INFINITY;
maxDepth = maxDepth - 1; // Still being POSITIVE_INFINITY
maxDepth > 0; // true
```

But it could be broken in the future, so this is changed at PR #90.